strawberryfields/apps/graph/utils.py

```diff
@@ -196,16 +220,27 @@ def c_1(clique: list, graph: nx.Graph):
     """Generates the set :math:`C_1` of nodes that are connected to all but one of the nodes in
     the input clique subgraph
 
-    The set :math:`C_1` is defined in :cite:`pullan2006phased`.
+    The set :math:`C_1` is defined in :cite:`pullan2006phased` and is used to determine outside
+    nodes that can be swapped with clique nodes to create a new clique.
+
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import utils
+        >>> import networkx as nx
+        >>> graph = nx.wheel_graph(5)
+        >>> subgraph = [0, 1, 2]  # these nodes form a clique
+        >>> utils.c_1(subgraph, graph)
+        [(1, 3), (2, 4)]
 
     Args:
         clique (list[int]): A subgraph specified by a list of nodes; the subgraph must be a clique
         graph (nx.Graph): the input graph
 
     Returns:
-       list[int]: A list of tuples ``[(i_clique, i), (j_clique, j),...,(k_clique, k)]``. Here
-       ``i,j,...,k`` are the nodes in :math:`C_1`, while ``i_clique, j_clique,...,k_clique`` are the
-        nodes in the clique they can be swapped with.
+       list[int]: A list of tuples. The first node in the tuple is the node in the clique and the
```

Comment:
Nice, looks a simpler explanation!
```suggestion
       list[tuple(int)]: A list of tuples. The first node in the tuple is the node in the clique and the
```